resources/texts/change_log.txt

```diff
@@ -25,8 +24,35 @@ We now support Apple M1 chips
 - Decrease resolution to remove some buffer underruns
 - Removed the setting Center Last from Wall Ordering
```

Comment:
This setting never appeared in any stable release. It was added in an alpha, and removed before the stable release.